src/index.html

```diff
@@ -0,0 +1,210 @@
+
+    </section>
+    <section id="about" class="about">
+        <div class="left">
+            <h2 class="about-me"> About Myself</h2>
+            <p class="intro1">
+                Hello I’m a software developer! I can help you build a
+                product , feature or website Look through some of my
+                work and experience! If you like what you see and have
+                a project you need coded, don’t hestiate to contact me.
+            </p>
+            <h2 class="intro2">LET’S CONNECT</h2>
+            <ul class="social">
+                <li class="social-icon">
+                    <img src="../assets/github.png" alt="github">
+                </li>
+                <li class="social-icon">
+                    <img src="../assets/Linkedin.png" alt="Linkedln">
+                </li>
+                <li class="social-icon">
+                    <img src="../assets/fingers.png" alt="two fingers">
+                </li>
+                <li class="social-icon">
+                    <img src="../assets/twitter.png" alt="twitter">
+                </li>
+
+                <li class="social-icon">
+                    <img src="../assets/m.png" alt="m">
+                </li>
+            </ul>
+            <div class="button-resume">
+                <button type="button" class="btn">
+                    Get my resume
+                </button>
+            </div>
+        </div>
+        <div class="right">
+            <div class="lang">
+
+            </div>
+            <ul class="language-list">
+                <li class="lang">
+                    <p>
+                        Languages
+                    </p>
+                    <img class="imglang" src="../assets/arrowdown.png" alt="arrow">
+                </li>
+                <li class="lang-skill">
+                    <img src="../assets/js.png" alt="js">
+                    <p>
+                        JavaScript
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="../assets/css.png" alt="html">
+                    <p>
+                        CSS
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="../assets/html.png" alt="css">
+                    <p>
+                        HTML
+                    </p>
+                </li>
+                <li class="lang">
+                    <p>
+                        FrameWorks
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+                <li class="lang">
+                    <p>
+                        Skills
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+            </ul>
+        </div>
+    </section>
```

Comment:
- [ ] As stated in the project requirement You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly make sure that There are 2 blocks **The left one takes 40% which is About me part with 4 to 5 lines of text, "LET'S CONNECT" section with social media icon-links , Lastly, there's a Get my Resume button aligned perfectly on the left**  and **the right one the remaining 60% of the page's width, which is Languages, Frameworks, and Skills information, drop-down menus, technology tags are grey squares that have the technology logo and below the technology, the name aligned to the left and the contents of each of them are aligned to the right**.
- The About Me section of the Figma design 
![Capture](https://user-images.githubusercontent.com/93598189/222535966-a938055c-3a71-4c02-b31e-4e3deee01fb2.PNG)

- Your Project Design for About Me section 
![Capture](https://user-images.githubusercontent.com/93598189/222536841-b0c8a58a-ebf0-4c69-afaf-0690e9e3f976.PNG)

Reply:
- [ ] Your attempt to reflect this is awesome, but I think you can make it better. Kindly read the suggestions again to really have feel of what the reviewer is saying. When you understand it, I believe you will know what needs to change. The order of arrangements - `flex` and spacing are very important to how how others view your page, and I am sure you want to charm recruiters with this. :wink: You are almost there!

Reply:
> * [ ]  As stated in the project requirement You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly make sure that There are 2 blocks **The left one takes 40% which is About me part with 4 to 5 lines of text, "LET'S CONNECT" section with social media icon-links , Lastly, there's a Get my Resume button aligned perfectly on the left**  and **the right one the remaining 60% of the page's width, which is Languages, Frameworks, and Skills information, drop-down menus, technology tags are grey squares that have the technology logo and below the technology, the name aligned to the left and the contents of each of them are aligned to the right**.
> * The About Me section of the Figma design
>   ![Capture](https://user-images.githubusercontent.com/93598189/222535966-a938055c-3a71-4c02-b31e-4e3deee01fb2.PNG)
> * Your Project Design for About Me section
>   ![Capture](https://user-images.githubusercontent.com/93598189/222536841-b0c8a58a-ebf0-4c69-afaf-0690e9e3f976.PNG)


Please, kindly ensure that your design matches as the previous reviewer have stated here. 😀